### PR TITLE
Potato hvac/use model folder in 2

### PR DESCRIPTION
**Pull request by potatoHVAC**
On a normal project any changes you make would go through a pier review process and get approved before merging. I'm going to use that process when adding to your repos. At some point I will work up a tutorial where we go through this process in detail. For now, I am using the pull request (PR) to make my changes more visible to you.

### CODE REVIEW

**Comment by potatoHVAC** [comment]
Typically at least one review is required before merging but we don't have those restrictions in place. Add a review in the top right and approve (or disapprove if you want).

**Comment by potatoHVAC** [comment]
Go ahead and hit the "Merge pull request" at the bottom of the page. Normally this step is done by whoever created the PR but what ever. 

Now you can use `git pull` to bring this new information into your local machine.

File: 2-Add-Two-Numbers.py
Comment (ghost) [inline 2-Add-Two-Numbers.py:12]: Should have known there was an easy way to do this. :-)

**Review by ghost — APPROVED** [review]
Looks good.

**Comment by ghost** [comment]
Do you delete branches after a merge?

**Pull request merged**
